apps/src/templates/Congrats.story.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Congrats from './Congrats';
+
+export default storybook => {
+  return storybook
+    .storiesOf('Congrats', module)
+    .addStoryTable([
+      {
+        name: 'Congrats - Applab',
+        description: `Congrats component if Applab tutorial completed`,
+        story: () => (
+          <Congrats
+            completedTutorialType="applab"
+          />
+        )
+      },
+      {
+        name: 'Congrats - old Minecraft',
+        description: `Congrats component if old Minecraft tutorial completed`,
```

Comment:
There are actually two old Craft tutorials: Adventurer (2015) and Designer (2016).  Do we need to differentiate between these as well, or only this year vs. previous years?